_includes/integrations/react-tech-ref.md

```diff
@@ -40,6 +41,27 @@ or with Yarn
 $ yarn add @tinymce/tinymce-react
 ```
 
+### Using TinyMCE React integration in a Bootstrap dialog
+
+To use TinyMCE React instances inside Bootstrap UI dialogs, add the following React effect to a component that renders with the editor:
+
+```jsx
+  useEffect(() => {
+    const handler = (e) => {
+      if (
+        $(e.target).closest(
+          ".tox-tinymce-aux, .moxman-window, .tam-assetmanager-root"
+        ).length
+      ) {
```

Comment:
```suggestion
      if (e.target.closest(".tox-tinymce-aux, .moxman-window, .tam-assetmanager-root").length) {
```
As Bootstrap 5 doesn't support IE 11 we can just use the native closest API.

Reply:
Made a version for bootstrap 4 that supports IE.